skyportal/tests/api/test_tag.py

```diff
@@ -9,90 +9,102 @@
 
 # --- Testing ObjTagOption API
 def test_get_tag(super_admin_token):
-    tag_to_create = {"tag_name": f"test_get_{random.randint(0, 100000)}"}
-    status, created_tag = api(
-        "POST", "objtagoption", data=tag_to_create, token=super_admin_token
-    )
+    tag_to_create = {"tag_name": f"TestTag{uuid.uuid4().hex}"}
+    status, _ = api("POST", "objtagoption", data=tag_to_create, token=super_admin_token)
+    assert status == 200
 
-    tag_name = created_tag["data"]["tag_name"]
-    tag_id = created_tag["data"]["id"]
+    status, _ = api("GET", "objtagoption", token=super_admin_token)
+    assert status == 200
 
-    status_by_id, _ = api("GET", f"objtagoption/{tag_id}", token=super_admin_token)
-    assert status_by_id == 200
 
-    status_by_id, _ = api("GET", f"objtagoption/999999", token=super_admin_token)
-    assert status_by_id == 404
+def test_add_tag_case_sensitive(super_admin_token):
+    unique_id = uuid.uuid4().hex
+    tag_to_create = {"tag_name": f"TagAdded{unique_id}"}
+    status, _ = api("POST", "objtagoption", data=tag_to_create, token=super_admin_token)
+    assert status == 200
+
+    # Verification that we can't create the same tag twice
+    status, _ = api(
+        "POST",
+        "objtagoption",
+        data={"tag_name": f"tagadded{unique_id}"},
+        token=super_admin_token,
+    )
+    assert status == 409
 
-    status_by_name, _ = api("GET", f"objtagoption/bonjour", token=super_admin_token)
-    assert status_by_name == 404
+    # Verification that we can't create a tag with a space
```

Comment:
Thanks for being so thorough! May I suggest a shorter way to write your tests though? If you have `N` number of characters you want to test, you might as well just have a list/array with those, and do your POST + assert in a loop over that array. That would make your test **much** easier to read and shorter, hence easier to maintain.